src/App.js

```diff
@@ -90,21 +102,54 @@ class Root extends React.Component {
             }
           }
         }
-        this.props.historyUpdate({'data': [wsData.history]});
+        wallet.newAddressHistory(wsData.history);
       } else {
         console.log('Websocket message not handled. Type:', wsData.type);
       }
     }
   }
 
   handleWebsocketStorage = (wsData) => {
-    if (wallet.loaded()) {
+    if (hathorLib.wallet.loaded()) {
       // We are still receiving lot of ws messages that are destined to the admin-frontend and not this wallet
       // TODO separate those messages
       console.log('Websocket message not handled. Type:', wsData.type);
     }
   }
 
+  /**
+   * Method called when WebSocket receives a message after loading address history
+   * We just check and save the version that was loaded and update redux data
+   *
+   * @param {Object} data Object with {'historyTransactions', 'addressesFound'}
+   */
+  addressesLoadedUpdate = (data) => {
+    // Update the version of the wallet that the data was loaded
+    localStorage.setItem('wallet:version', VERSION);
```

Comment:
I think we just write it to the `localStorage` to check whether the database is old when a new version is released.

Reply:
I need to set after the `App.js` `componentDidMount` method because we need to verify the current wallet version with the last one to see if they are still compatible. That's why every request to load addresses I set it, to guarantee the storage has this version always updated